src/ImageSharp/Colors/Color.cs

```diff
@@ -112,6 +112,9 @@ public Color(uint packed)
             this.packedValue = packed;
         }
 
+        /// <inheritdoc />
+        public BulkPixelOperations<Color> BulkOperations => new BulkPixelOperations<Color>();
```

Comment:
shouldn't these be reading a static instance of `BulkPixelOperations<T>` ? otherwise it'll be really easy to allocate multiple instances from a public API.

The way I see it is it should be more like this
``` c#
private static readony BulkPixelOperations<Color> PrivateBulkOperations = new BulkPixelOperations<Color>();

public BulkPixelOperations<Bgra5551> BulkOperations => PrivateBulkOperations;
```

That way we still get to use the hack to allow for swapping out to more efficiant versions but we wont accidentally allocate a new `BulkPixelOperations<T>` every time you allocate the property on this or any other instance of the struct.

This obviously applies to all `TColor`s